wallet-new/server/Cardano/Wallet/API/V1/LegacyHandlers/Wallets.hs

```diff
@@ -57,9 +58,17 @@ newWallet NewWallet{..} = do
                               <*> pure 0
     let walletInit = V0.CWalletInit initMeta backupPhrase
     single <$> do
-        v0wallet <- newWalletHandler newwalOperation spendingPassword walletInit
+        v0wallet <- newWalletHandler newwalOperation spendingPassword walletInit `catch` rethrowDuplicateMnemonic
```

Comment:
this line also looks too long

Reply:
Ps, I am bad at breaking some lines as it diminishes readibility. Do you leave some lines over 80chars or do you try to fit all in 80 chars ? (or maybe you do it automatically)

_ I am not talking about the line above though, just in general_

Reply:
Its definitely a taste/style thing -- I try to keep lines under 80 characters always. There are studies that have shown that wider columns make it harder to read, and there's a reason most print has fairly narrow columns. 

For my own workflow, I like to have readable font + my terminal split in two vertically, so that I can have a browser/tests/docs/etc on the right hand side. 80 characters is near the sweet spot for that I find. For bigger or multiple-monitor setups, I'll have three vertical splits!

For this specific line, I'd write:

```haskell
v0wallet <- newWalletHandler newwalOperation spendingPassword walletInit 
        `catch` rethrowDuplicateMnemonic
```